paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -177,7 +180,7 @@ internal class DefaultFlowController @Inject internal constructor(
     ) {
         configure(
             mode = PaymentSheet.InitializationMode.PaymentIntent(paymentIntentClientSecret),
-            configuration = configuration,
+            configuration = configuration ?: PaymentSheet.Configuration.default(viewModel.getApplication()),
```

Comment:
This will make it harder for us to convert this to a non-Android ViewModel. Can we solve this differently? Maybe the `default()` method just takes a `label` argument?

Reply:
Yep, it'll make it harder. But flow controller already has access to context. I just pulled it from here (because it was easy) instead of adding a new property to the flow controller that's explicitly a context. But can do that if you prefer.